src/core_plugins/kibana/public/home/components/recently_accessed.less

```diff
@@ -7,6 +7,11 @@
 .recentlyAccessedItem {
   overflow: hidden;
   max-width: 300px;
+
+  .euiToolTipAnchor {
+    overflow: hidden;
```

Comment:
I dislike nested CSS rules, as they have been the source of much past pain. I'd consider putting a scenario-specific class on the elements, and applying styles to that... That said, this is a reasonable solution, especially if specificity is required to for the override to work properly.

Reply:
There is no way to attach a custom class to the anchor element. The `className` prop is passed to the tooltip popover and not the wrapping anchor span.

https://github.com/elastic/eui/blob/master/src/components/tool_tip/tool_tip.js
```
  render() {
    const {
      children,
      className,
      content,
      title,
      ...rest
    } = this.props;

    const classes = classNames(
      'euiToolTip',
      positionsToClassNameMap[this.state.calculatedPosition],
      className
    );

    let tooltip;
    if (this.state.visible) {
      tooltip = (
        <EuiPortal>
          <EuiToolTipPopover
            className={classes}
            style={this.state.toolTipStyles}
            positionToolTip={this.positionToolTip}
            title={title}
            id={this.state.id}
            role="tooltip"
            {...rest}
          >
            {content}
          </EuiToolTipPopover>
        </EuiPortal>
      );
    }

    const anchor = (
      <span
        ref={anchor => this.anchor = anchor}
        className="euiToolTipAnchor"
      >
        {cloneElement(children, {
          onFocus: this.showToolTip,
          onBlur: this.hideToolTip,
          'aria-describedby': this.state.id,
          onMouseOver: this.showToolTip,
          onMouseOut: this.onMouseOut
        })}
      </span>
    );

    return (
      <Fragment>
        {anchor}
        {tooltip}
      </Fragment>
    );
  }
}

```